pipelines/templates/stages/build.yml

```diff
@@ -14,10 +14,15 @@ stages:
           - bash: az bicep build -f deploy/azuredeploy.bicep --outdir /tmp
             displayName: Compile Bicep
           - bash: |
-              cmp --silent /tmp/azuredeploy.json deploy/azuredeploy.json
+              diff <(tail -n +10 /tmp/azuredeploy.json) <(tail -n +10 deploy/azuredeploy.json)
```

Comment:
is there a chance that something new will be added to the metadata down the road that this could gloss over?